dev/user.clj

```diff
@@ -0,0 +1,110 @@
+(ns user
+  (:require [neo4clj.client :as client]))
+
+#_(def conn (client/connect "bolt://localhost:7687" "neo4j" "password"))
+
+(defn test-basic-node-logic
+  [conn]
+  (let [node (client/create-node! conn {:ref-id "n"
+                                        :labels [:person]
+                                        :props {:first-name "Neo" :last-name "Anderson"}})]
+    (println "Nodes after creation:" (count (client/find-nodes conn {:ref-id "n" :labels [:person]})))
```

Comment:
Maybe it would be a good help to include the expected number in the string for `println` if someone without as much experience runs the script (or you leave it for 6 months and come back). Kinda like test cases. But it makes it easy to quickly spot things that doesn't match.

It allows for another person to run this on every PR on their own machine and verify they get the same result as the expected.